Dockerfile

```diff
@@ -5,7 +5,7 @@ MAINTAINER Casey Davenport <casey@tigera.io>
 # Install libc (to run calicoctl) and various build tools to build the vendored packages.
 ENV GLIBC_VERSION 2.23-r1
 RUN apk add --update bash git iproute2 curl make gcc alpine-sdk linux-headers && \
-  apk add --update-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing etcd && \
+  apk add --update-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing etcd consul && \
```

Comment:
I thought that this is used only for testing and at the moment of writing etcd was made same. Difference is 22.1 Mb. I'll remove this dependency and rewrite some test code :)